autoload/ale/fixers/importjs.vim

```diff
@@ -9,7 +9,7 @@ function! ale#fixers#importjs#ProcessOutput(buffer, output) abort
 endfunction
 
 function! ale#fixers#importjs#Fix(buffer) abort
-    let l:executable = ale#Var(a:buffer, 'js_importjs_executable')
+    let l:executable = ale#Var(a:buffer, 'javascript_importjs_executable')
```

Comment:
Update the variable above too, and in the documentation, if it's documented there. If the option is documented, write `get(g:, 'ale_js_importjs_executable', 'importjs')` above so we won't break anything for people who are using the old variable name.

Reply:
It is documented as:
```
===============================================================================
importjs                                              *ale-javascript-importjs*

g:ale_javascript_importjs_executable     *g:ale_javascript_importjs_executable*
                                         *b:ale_javascript_importjs_executable*
  Type: |String|
  Default: `'importjs'`


===============================================================================
```

So the code is not in sync with the documentation or vice versa.
Do you still want the 'get(...)' for backwards compatibility?

Reply:
Nah, if it's not documented, then don't worry about it.